pallets/staking-rewards/src/lib.rs

```diff
@@ -326,6 +329,26 @@ pub mod pallet {
 
             Ok(())
         }
+
+        #[pallet::call_index(3)]
+        #[pallet::weight(<T as Config>::WeightInfo::align_supply())]
+        pub fn align_supply(origin: OriginFor<T>, target: BalanceOf<T>) -> DispatchResult {
+            ensure_root(origin)?;
```

Comment:
In this pallet we have `RefillOrigin` (which is better be renamed as `DepositOrigin`) and `WithdrawOrigin` for respective operations. Maybe we can introduce another generalized origin one for this type of operation, too, to have more freedom in setting permissions here, as well?